backend/src/entities/token.rs

```diff
@@ -80,7 +80,7 @@ use backend_service::config::HEADER_TOKEN;
 use futures::future::{ready, Ready};
 
 impl FromRequest for Token {
-    type Config = ();
+    //type Config = ();
```

Comment:
The latest beta version of actix-web has removed this. I'll fix this by updating cargo.toml with actix-web = "4.0.0-beta.11"